api/v1alpha1/clusterqueue_types.go

```diff
@@ -156,6 +157,10 @@ type QueueingStrategy string
 const (
 	// StrictFIFO means that workloads are sorted strictly by creation time.
 	StrictFIFO QueueingStrategy = "StrictFIFO"
+
+	// BestEffortFIFO means that workloads are sorted based on creation time
+	// with event-based re-queueing.
```

Comment:
"with event-based re-queueing." is an implementation detail, can we describe the semantics in a different way? like: "Workloads will be admitted based on their creation time, however older workloads that can't be admitted will not block newer ones that can"

Reply:
also add a comment that this is not yet implemented and reference the issue

Reply:
Updated.